Resources/Prototypes/Entities/Clothing/Eyes/hud.yml

```
@@ -12,6 +12,7 @@
   abstract: true
   noSpawn: true
   components:
+  - type: ShowDeadWithoutMind
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)